src/Data/List/Properties.agda

```diff
@@ -758,6 +758,10 @@ length-take zero    xs       = refl
 length-take (suc n) []       = refl
 length-take (suc n) (x ∷ xs) = cong suc (length-take n xs)
 
+take-one-more : {ℓ : Level} {X : Set ℓ} {m : ℕ} (x : Fin m) (f : Fin m → X) →
```

Comment:
This should definitely have `tabulate` in the name somewhere. How about `take-suc-tabulate`?